plugin/src/main/java/org/opensearch/ml/action/controller/CreateControllerTransportAction.java

```diff
@@ -99,7 +99,8 @@ protected void doExecute(Task task, ActionRequest request, ActionListener<MLCrea
 
         try (ThreadContext.StoredContext context = client.threadPool().getThreadContext().stashContext()) {
             ActionListener<MLCreateControllerResponse> wrappedListener = ActionListener.runBefore(actionListener, context::restore);
-            mlModelManager.getModel(modelId, null, excludes, ActionListener.wrap(mlModel -> {
+            // TODO: Add support for multi tenancy
```

Comment:
This Controller actions will be disabled for multi-tenancy. @b4sjoo is working on this.